jetty-server/src/main/java/org/eclipse/jetty/server/LowResourceMonitor.java

```diff
@@ -247,112 +286,151 @@ public void setMaxLowResourcesTime(int maxLowResourcesTimeMS)
         _maxLowResourcesTime = maxLowResourcesTimeMS;
     }
 
-    @Override
-    protected void doStart() throws Exception
+    @ManagedAttribute("The maximum memory (in bytes) that can be used before low resources is triggered.  Memory used is calculated as (totalMemory-freeMemory).")
```

Comment:
the LowResourceCheck instances themselves should also be @ManagedObjects if they have configuration that can be @ManagedAttributes.